interfaces/builtin/all_test.go

```diff
@@ -115,10 +115,10 @@ type seccompDefiner4 interface {
 }
 
 type systemdDefiner1 interface {
-	SystemdConnectedPlug(spec *systemd.Specification, plug *interfaces.Plug, plugAttrs map[string]interface{}, slot *interfaces.Slot, slotAttrs map[string]interface{}) error
```

Comment:
Can you please move the old definitions to "known old APIs" that we detect and error on?

Reply:
Done.